tox.ini

```diff
@@ -4,11 +4,17 @@ envlist = py{36,37}
 [testenv]
 commands=
     pytest --durations=5 --cov=libertem --cov-report=term --cov-report=html --cov-report=xml {posargs:tests/}
-    pytest --doctest-modules src/libertem/common/
+    # win_tweaks.py depends on modules that are only available on Windows
+    pytest --doctest-modules --ignore=src/libertem/win_tweaks.py src/libertem/
 deps=
     -rtest_requirements.txt
 extras=
     hdbscan
+    hdfs
+setenv=
+    # Somehow the conftest.py file in src/libertem leads to a collision
+    # See https://github.com/pytest-dev/pytest/issues/2042
+    PY_IGNORE_IMPORTMISMATCH=1
```

Comment:
People discussed this *at length* in the referenced issues and this is what they came up with. I am updating the comment since it affects *all* files that are under src/libertem